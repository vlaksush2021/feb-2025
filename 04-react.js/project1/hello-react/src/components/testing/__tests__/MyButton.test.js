import React from 'react';
import renderer  from 'react-test-renderer'
import MyButton from '../MyButton';

describe('button tests', () => {

    test.skip('button test1', () => {
        const tree = renderer.create(<MyButton label="SubmitSubmit Me" />).toJSON();
        console.log(tree);
        expect(tree).toMatchSnapshot()
    })
})