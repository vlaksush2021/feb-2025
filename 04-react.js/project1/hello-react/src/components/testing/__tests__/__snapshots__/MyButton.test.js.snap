// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button tests button test1 1`] = `
<button>
  SubmitSubmit Me
</button>
`;
